src/node/node_info_network.h

```diff
@@ -137,4 +161,33 @@ namespace ccf
         PRIMARY_RPC_INTERFACE, std::move(primary_interface));
     }
   }
-}
\ No newline at end of file
+}
+
+FMT_BEGIN_NAMESPACE
+template <>
+struct formatter<ccf::Authority>
+{
+  template <typename ParseContext>
+  auto parse(ParseContext& ctx)
+  {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext>
+  auto format(const ccf::Authority& authority, FormatContext& ctx)
+    -> decltype(ctx.out())
+  {
+    switch (authority)
+    {
+      case (ccf::Authority::NODE):
+      {
+        return format_to(ctx.out(), "Node");
+      }
+      case (ccf::Authority::SERVICE):
+      {
+        return format_to(ctx.out(), "Service");
+      }
```

Comment:
A default throw is probably prudent here.

Reply:
It's not needed. With new enum members you would a compile error: `node_info_network.h:180:13: error: enumeration value 'FOO' not handled in switch [-Werror,-Wswitch]`